pkg/kubelet/kubelet_node_status_test.go

```diff
@@ -1483,6 +1518,10 @@ func TestTryRegisterWithApiServer(t *testing.T) {
 	}
 
 	for _, tc := range cases {
+
```

Comment:
I think we should wrap these statements in function closures, otherwise `defer` will only be executed on the function is finished 
```
func() {
... 
}()
```

Reply:
Done, for this to work, each test had to be subtest. made it so.